src/components/ingestion/FileIngestionEngine.js

```diff
@@ -0,0 +1,33 @@
+import fs from 'fs'
+import AbstractIngestionEngine from '%src/components/ingestion/AbstractIngestionEngine'
+
+/**
+ * The FileIngestionEngine handles processing a video file. It is a concrete
+ * implementation of AbstractIngestionEngine, which reads in a file,
+ * converts it into an MPEG-TS stream represented as a series of Payloads,
+ * and then pipes that to Kafka.
+ *
+ * @param  {string}  path The path of the file to be ingested
```

Comment:
I think Dan mentions this already (it's annoying that I'm not seeing his review comments in this tab), but yeah I thiiiink we JSDoc the constructor and not the top of the class? (Update: yes!)

Reply:
Okay!

Reply:
Done.